packages/form-js-editor/src/render/components/properties-panel/PropertiesPanel.js

```diff
@@ -35,11 +36,11 @@ function getGroups(field, editField) {
     GeneralGroup(field, editField)
   ];
 
-  if (type === 'radio' || type === 'select') {
+  if (type === 'radio' || type === 'select' || type === 'checklist') {
     groups.push(ValuesGroup(field, editField));
   }
 
-  if (INPUTS.includes(type) && type !== 'checkbox') {
+  if (INPUTS.includes(type) && type !== 'checkbox' && type != 'checklist') {
```

Comment:
Could we move these to `Util` as well? I added something similar via the [properties panel migration](https://github.com/bpmn-io/form-js/pull/254/files#diff-f39d8b1f36be0837b1b467eed602c62a2ed76e484e25220efc988f5753d27fc4R45). Saying that, I think it's okay to refactor this via https://github.com/bpmn-io/form-js/pull/254.

Reply:
I think we should do this later, reasoning being that there's currently 3 PRs that modify these and it'll be simpler to fix in one fell swoop than re-arrange the file structure at this stage.